digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/csvDataImporter/LanesCsvImporter.scala

```diff
@@ -153,7 +153,10 @@ class LanesCsvImporter(roadLinkServiceImpl: RoadLinkService, eventBusImpl: Digir
     (optTrack, optLane) match {
       case (Some(track), Some(lane)) =>
         (Track.apply(track.toInt), lane.charAt(0).getNumericValue) match {
-          case (Track.RightSide, 2) | (Track.LeftSide, 1)  => (List(s"Wrong lane number for the track given"), List())
+          case (Track.RightSide, 2) | (Track.LeftSide, 1)  =>
+            (List(s"Wrong lane number for the track given"), List())
+          case (_, _) if lane.charAt(1).getNumericValue == 1 =>
```

Comment:
```suggestion
           case (_,1) => (List(s"Not allowed to import main lanes"), List())
```
vois olla yksinkertaisempaa

Reply:
lane.charAt(1) vähän ihmetyttää kun ylemmässä otetaan lane.charAt(0)

Reply:
Muuten kyllä, mutta toi kattoo väärää numeroa. (_, 1) tuossa toi 1 on kaksi numeroisen kaistan eka numero eli tieosoitteen kasvusuunta. Tässä me ollaan kiinnostuneita siitä onko vika numero 1 eli ollaanko tuomassa pääkaistaa.

Reply:
edellinen kehittäjä laittanut ehkä vähän liian monta case match peräkkäin, en nyt tiedä on pyramiidi case match paremppi kuin pyramiidi if lauseita tai looppeja

Reply:
ok